app/controllers/admin/editions_controller.rb

```diff
@@ -207,9 +207,13 @@ def get_layout
     end
   end
 
-  def fetch_version_and_remark_trails
-    @document_remarks = Document::PaginatedRemarks.new(@edition.document, params[:remarks_page])
-    @document_history = Document::PaginatedHistory.new(@edition.document, params[:page])
+  def fetch_version_and_remark_trails(preview_design_system: false)
+    if preview_design_system
+      @document_history = Document::PaginatedTimeline.new(document: @edition.document, page: params[:page] || 1)
```

Comment:
Is this param name correct?

```
page: params[:page]
```

In the views, we pass `param_name: 'history_page'` to the helper method:

https://github.com/alphagov/whitehall/blob/ea2b5baec0a081d59b4d0b6b5ab463311edd9f49/app/views/admin/editions/show/_sidebar.html.erb#L21

Although I might be misinterpreting what that helper param does.